PaulasWeatherApp/PaulasWeatherApp/Model/APIModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  APIModel.swift
+//  PaulasWeatherApp
+//
+//  Created by Stef, Paula on 7/26/22.
+//
+
+import Foundation
+
+class APIModel {
+    private static let key = "5a0bc1288e1afcee4cf0059dbb4b654c"
+    
+    static func getWeatherData(lat: Float, lon: Float, completionHandler: @escaping (_ data: Data?, _ response: URLResponse?, _ error: Error?) -> Void) {
+        let url = URL(string: "https://api.openweathermap.org/data/2.5/onecall?lat=\(lat)&lon=\(lon)&units=imperial&exclude=minutely,icon&appid=\(self.key)")
```

Comment:
The values shouldn't be fetched for the imperial metric system. Also you should fetch the data for the current time. If you take a look here: https://openweathermap.org/api/one-call-3 you'll see that the endpoint can have a format like the one below:
https://api.openweathermap.org/data/3.0/onecall/timemachine?lat={lat}&lon={lon}&dt={time}&appid=[{API key}](https://home.openweathermap.org/api_keys)